src/a11yCore.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+/**
+ * a11yCore module - used as a namespace.
+ * @module a11yCore
```

Comment:
This module acts as a wrapper and effectively namespace for the color module. There might be a better implementation, or a better API, but wanted to match what's in ADT as closely as possible initially.